bin/check-graylog-buffers.rb

```diff
@@ -37,6 +37,12 @@
 require 'rest-client'
 
 class CheckGraylogBuffers < Sensu::Plugin::Check::CLI
+  option :protocol,
+         description: 'Protocol for connecting to Graylog',
+         short: '-proto',
```

Comment:
I am actually not sure if short is ok with multi char short args. Have you tested this? Usually short options single chars that can be chained together like `tar -xvf` that is not a single arg but actually 3 of them.